src/Generator/Synapse/ImportFull/FromAbsGenerator.php

```diff
@@ -90,6 +90,11 @@ public function generate(array $columns, array $primaryKeys = []): array
                 Utils::getUniqeId('destTableName'),
                 'tableName',
             ),
+            'timestamp' => new ReplaceToken(
+                '\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}',
+                'q(timestamp)',
```

Comment:
to `q()` nepatri sem, ale do `Replace.php`, podobne jako tady https://github.com/keboola/app-custom-query-manager/pull/5/files#diff-b1bd6d610b69c67dfda9001000bf9ac5fab0feba5f96608562ad456cf58cb0b2R96

Reply:
Proč?

Reply:
protoze tohle `TYPE_MATCH_AS_VALUE_REGEX` rika, ze je to `value` a tebe nemusi zajimat, jak se osetruje. V tom `Replace.php` jsou i `q()` i `id()` u ostatnich typu

Reply:
Takže jen semanticky - prostě pak bude muset případně být TYPE_MATCH_AS_IDENTIFIER_REGEX a tam bude id(). Ale klidně by se to dalo řešit nastavením přímo v tom Generatoru, ne? 

Každopádně - upraveno.